base/libgit2/libgit2.jl

```diff
@@ -161,7 +195,15 @@ function make_payload{P<:AbstractCredentials}(payload::Nullable{P})
     Ref{Nullable{AbstractCredentials}}(payload)
 end
 
-""" git fetch [<url>|<repository>] [<refspecs>]"""
+"""
+    fetch(repo::GitRepo, remote::AbstractString="origin", remoteurl::AbstractString="", refspecs=AbstractString[], payload=Nullable{AbstractCredentials}())
+
+Equivalent to `git fetch [<url>|<repository>] [<refspecs>]`.
+Fetches updates from the `remote` upstream of `repo`, looking at `remoteurl`.
```

Comment:
Might be nice as "Fetches updates from the upstream `remote` with URL `remoteurl` of the repository `repo`."